products/BellHybrid/apps/application-bell-relaxation/model/RelaxationSongsRepository.cpp

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) 2017-2023, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+#include "RelaxationSongsRepository.hpp"
+#include <module-db/queries/multimedia_files/QueryMultimediaFilesGetLimited.hpp>
+#include <module-db/queries/multimedia_files/QueryMultimediaFilesCount.hpp>
+#include <AsyncTask.hpp>
+#include <vector>
+
+namespace
+{
+    std::uint32_t calculateOffset(const std::uint32_t offset, const std::uint32_t filesCount)
```

Comment:
These consts are not really needed here

Reply:
We should discuss it with a team :D 
I think this is a good practice to keep this a constant. It protects you from unwanted updates of the value.
We can also consider passing as a const& even for ints. The cost and time are the same :D

Reply:
Discussed on slack.